client/reader/stream/style.scss

```diff
@@ -646,7 +646,7 @@
 			}
 
 			@include breakpoint-deprecated( "<660px" ) {
-				margin-inline: 16px 16px;
+				margin-inline: 17px 17px;
```

Comment:
Not a blocker by any stretch of the imagination, but a benefit of a mixin system or variables would be you could type out something like `$grid-unit-20 + $border-width`, and get the same result, which would be easier to read code-wise, without needing to comment.